src/main/steps/applicant2Sequence.ts

```diff
@@ -196,6 +198,18 @@ const postSubmissionSequence: Step[] = [
     url: CHECK_CONDITIONAL_ORDER_ANSWERS_URL,
     getNextStep: () => HUB_PAGE,
   },
+  {
+    url: FINALISING_YOUR_APPLICATION,
+    getNextStep: data => (data.state === State.FinalOrderOverdue ? EXPLAIN_THE_DELAY : HUB_PAGE),
+  },
+  {
+    url: EXPLAIN_THE_DELAY,
+    getNextStep: () => HUB_PAGE,
+  },
+  {
+    url: PROVIDE_INFORMATION_TO_THE_COURT,
```

Comment:
how comes this is being added? I am not seeing anything in this pr related to this page.

Reply:
I thought this was scenario 7, where continuing AFTER one year from final order due date when the other applicant did not apply. And the state us FinalOrderOverdue.

Reply:
Removed that step as its not in scope.